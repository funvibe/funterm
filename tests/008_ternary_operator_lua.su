lua.x = 5
lua.result = lua.x > 10 ? "greater than 10" : "less than or equal to 10"
lua.print(lua.result)