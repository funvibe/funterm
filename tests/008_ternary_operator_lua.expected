less than or equal to 10